src/test/java/core/basesyntax/service/impl/DataValidatorServiceImplTest.java

```diff
@@ -0,0 +1,152 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.service.DataValidatorService;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class DataValidatorServiceImplTest {
+    private static final List<String> VALID_DATA = Arrays.asList("type,fruit,quantity",
+            "b,banana,20",
+            "s,banana,100",
+            "p,banana,13",
+            "r,apple,10");
+    private final DataValidatorService dataValidatorService =
+            new DataValidatorServiceImpl();
+
+    @Test
+    public void test_DataValidator_DataIsValid_Ok() {
+        List<String> mutableData = new ArrayList<>(VALID_DATA);
+        assertTrue(dataValidatorService.isDataValid(mutableData));
+    }
+
+    @Test
+    public void test_DataValidator_DataIsNull_NotOk() {
+        List<String> mutableData = null;
```

Comment:
you have this variable in each test, move this to the class field and init in each test